README.md

```diff
@@ -1,106 +1,87 @@
+|-------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Databases Service | Write as the "Database Service" (as a proper noun). The Databases Service is referred to as "Databases Service" rather than "Database Service".                                                              |
+| Database          | As of 0.15, the Databases Service allows you to create multiple, individual databases. Use "a database" or "databases" as uncapitalized nouns, which differentiates them from the "Database Service". |
+| Collection        | Use "collection" as a noun (no capitalization), use as "a collection" or "collections".  Avoid using similar terms like "table".                                                                      |
+| Document          | Use "document" as a noun (no capitalization), use as "a document" or "documents".  Avoid using similar terms like "entry" or "row".                                                                   |
+| Attribute         | Use "attribute" as a noun (no capitalization), use as "a attribute" or "attributes".  Avoid using similar terms like "column" or "key/value".                                                         |
+| Index             | Use "index" as a noun (no capitalization), use as "a index" or "indexes". Use "indexes" instead of indices in a DB related context.                                                                   |
+| Permissions       | Use "Permissions" to describe the definition of read and write access to a resource. The definition is by User ID, Team ID, or Role. |
+| Permission Level             | Use "Permission Level" to describe the scope of defined permissions. Permission can be defined per resource (for each file) or per group of resource (for each file in a bucket).|
+
+#### Storage
+
+| **Term**        | **Suggested Usage**                                                                                                                          |
+|-----------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+| Storage Service | Write as the "Storage Service" (as a proper noun).                                                                                            |
+| Bucket          | Write as "a bucket" or "buckets", not a proper noun.                                                                                          |
+| File            | Write as "a file" or "files", not a proper noun.                                                                                              |
```

Comment:
Should we include terms related to big files support such as chunks and stream download?